libs/design/docs/theming.md

```diff
@@ -0,0 +1,27 @@
+### Step 1: Add the path to your angular configuration file.
+So inside your project `angular.json` insert the following inside `projects.project-name.architect.build.options`
```

Comment:
> So inside your project `angular.json` insert the following inside `projects.project-name.architect.build.options`

Within your project's `angular.json` append the following to the key `projects.project-name.architect.build.options`